lib/arrow_web/router.ex

```diff
@@ -79,4 +80,11 @@ defmodule ArrowWeb.Router do
   # scope "/api", ArrowWeb do
   #   pipe_through :api
   # end
+
+  if Mix.env() == :dev do
```

Comment:
question: is this safe to run in prod? Note https://hexdocs.pm/mix/1.14.5/Mix.html#env/0 says: 

> This function should not be used at runtime in application code (as opposed to infrastructure and build code like Mix tasks). Mix is a build tool and may not be available after the code is compiled (for example in a release).
>
> To differentiate the program behavior depending on the environment, it is recommended to use application environment through [Application.get_env/3](https://hexdocs.pm/elixir/1.14/Application.html#get_env/3).

It may be better to add a new config key like `enable_live_dashboard`, set it to `false` as the default in `config.exs`, and then set it to `true` in `dev.exs`. Alternatively we could make it available in prod as well, and restrict it to users in the proper group.
```suggestion
  if Application.compile_env!(:arrow, :enable_live_dashboard) do
```

Reply:
> It is safe to run in prod but I didn't set that up as part of this PR.

I meant "is it safe to run `Mix.env` in prod?", seeing as it's a build tool as the docs mention. I guess maybe in this case because it's being run at compile time, Mix is guaranteed to be available.